### PR TITLE
[Magiclysm] Demon Spider Melee Dice Balance/Fix

### PR DESCRIPTION
**Pull request by ianlintner**
#### Summary

```SUMMARY: Mods "Magiclysm update demon spider melee dice from 20 to 4 & damage sides to 12"```

#### Purpose of change

This maybe a potential off by 10 or additional "0"

#### Describe the solution

Bring melee dice in line with similar monsters "4" and 12 dice

#### Describe alternatives you've considered

Someone from Magiclysm should review this to confirm it should not be 20. Other "high level" melee focused monsters seem to max out at 5 melee dice in this mod. /data/mods/Magiclysm/monsters/dragon.json

### CODE REVIEW

**Comment by KorGgenT** [comment]
while 20d8 is indeed high, 2d8 is laughably low. demon spiders are supposed to be end-game monsters, not things you go melee with a cudgel

**Comment by ZhilkinSerg** [comment]
12?

**Comment by ianlintner** [comment]
https://github.com/CleverRaven/Cataclysm-DDA/issues/45127 was this merged elsewhere 

Demon spiders and the queen are also casters and can do plenty of damage without melee not including special attack modifers. 


Even at 6+ dice I think this maybe one of the highest damage output melee enemies (without casting) in the game correct me if I am wrong.

I could see making them in line with dragons.

**Comment by ianlintner** [comment]
Maybe something 
"melee_dice": 4,
"melee_dice_sides": 12,

File: data/mods/Magiclysm/monsters/demon_spider.json [hunk 1]
```diff
@@ -77,8 +77,8 @@
     "aggression": 50,
     "morale": 100,
     "melee_skill": 9,
-    "melee_dice": 2,
-    "melee_dice_sides": 8,
+    "melee_dice": 4,
+    "melee_dice_sides": 12,
```
Comment (ianlintner) [inline data/mods/Magiclysm/monsters/demon_spider.json]: Updated per feedback on 2 dice being too weak.

**Comment by KorGgenT** [comment]
so i've been thinking about this for a really long time, and i don't think i'll accept his PR right now without concrete melee balance data. you're really not supposed to melee these monsters, and the only thing you've said about it is to "bring it in line" with other monsters, but the demon spider brood really are supposed to be end-game monsters.

**Pull request closed without merging**